pkg/build/admission/strategyrestrictions/admission_test.go

```diff
@@ -250,6 +254,15 @@ func testBuild(strategy buildapi.BuildStrategy) *buildapi.Build {
 	}
 }
 
+func asV1Build(in *buildapi.Build) *buildapiv1.Build {
+	out := &buildapiv1.Build{}
+	err := kapi.Scheme.Convert(in, out, nil)
+	if err != nil {
+		panic(err)
+	}
+	return out
+}
```

Comment:
also seems like a helper.

Reply:
Single point of use is here.

Reply:
you realize i'm probably about to have to do this everywhere else as part of converting to v1 apis, right?